code/API_definitions/EdgeCloud_LcM.yaml

```diff
@@ -0,0 +1,658 @@
+                    interfaceName:
+                      type: string
+                      pattern: ^[a-z][a-z0-9]{3}$
+                      description:  Interface Name. Required only if application has to be attatched to a network other then default.
+                    visibilityType:
+                      description: Defines whether the interface is exposed to outer world or not i.e., external, or internal.
+                                          If this is set to "external", then it is  exposed to external applications otherwise it is exposed 
+                                          internally to edge application components within edge cloud. When exposed to external world, 
+                                          an external dynamic port is assigned for UC traffic and mapped to the internal containerPort
+                      type: string
+                      enum:
+                        - VISIBILITY_EXTERNAL
+                        - VISIBILITY_INTERNAL
+                minItems: 1                
+              numOfInstances:
+                type: integer
+                format: int32
+                description: Number of component instances to be launched. 
+              restartPolicy:
+                type: string
+                enum:
+                  - RESTART_POLICY_ALWAYS
+                  - RESTART_POLICY_NEVER
+                description: How the platform shall handle component failure                        
+      required:
+        - name
+        - virtType
+        - fileName
+        - repository
+        - componentSpec
+    CpuArchType:
+          type: string
+          enum:
+            - x86_64
+            - arm_64
+          description: CPU Instruction Set Architecture (ISA) E.g., Intel, Arm etc. 
+    ProvisionedApp:
+      description: Information about the provisioned application
+      type: object
+      properties:
+        appInstanceId:
+          $ref: '#/components/schemas/AppInstanceId'
+        edgeResourceName:
+          $ref: '#/components/schemas/EdgeResourceName'
+    EdgeResourceName:
+      description: Edge Resource Name - an identifier for an edge reource in the operator domain 
+      type: string
+ 
+    ErrorInfo:
+      type: object
+      properties:
+        status:
+          type: integer
+          description: HTTP status code returned along with this error response
+        code:
+          type: string
+          description: Code given to this error
+        message:
+          type: string
+          description: Detailed error description
+      required:
+        - status
+        - code
+        - message
+    FileId:
+      type: string
+      format: uuid
+      description: A globally unique identifier assosiated with the image file. Originating OP generates this identifier when file is uploaded over NBI.
+ 
+    MecDetails:
+      type: object
+      required:
+        - geolocation
+      properties:
+        edgeResourceName: 
```

Comment:
I suggest renaming the "edgeResourceName" field to just "name", as it's already scoped within the MecDetails object. This is consistent with other objects that just have a "name" field.

Reply:
We suggest to have it as it is so to clarify which name the response is refering to (there are more schemas and responses using parameters like name). Also as this is used in the response of a GET method, we find it more clarfying. But we can discuss it.

Reply:
As simple edge discovery (SED) API may become part of this file itself, we may need to harmonize that if the conceptual meaning of edgeResourceName for this API is same as in SED then we should have a one definition of this parameter

Reply:
We have rename the entity of edgeResourceName for edgeCloudNodeName, this is something we have to discuss to harmonize common entities in TI, SED and MVP API.